### PR TITLE
Storm's Eye module, and some quality updates

### PR DESCRIPTION
**Pull request by NamelessFaceless**
- Added a Storm's Eye uptime module
- Fixed the Rage suggestion into a Tiered Suggestion, and fixed the severity.
- Added a Inner Release 'rush' mechanic. It doesn't yell at the player if the fight ended in the middle of Inner Release anymore.
- Changed the wording in some suggestions.

### CODE REVIEW

File: src/parser/jobs/war/StormsEye.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const STORMS_EYE_DURATION = 30000
+const STORMS_EYE_BUFFER = 10000
+
+export default class StormsEye extends Module {
+	static handle = 'stormseye'
+	static dependencies = [
+		'checklist',
+		'combatants',
+		'invuln',
+		'cooldowns',
```
Comment (MikeMatrix) [inline src/parser/jobs/war/StormsEye.js]: Is the cooldowns dependency even used? 👀

Comment (NamelessFaceless) [reply src/parser/jobs/war/StormsEye.js]: Was in an earlier version, not anymore. xd

File: src/parser/jobs/war/InnerRelease.js [hunk 2]
```diff
@@ -123,22 +143,39 @@ export default class InnerRelease extends Module {
...
+			const numUpheavals = ir.casts.filter(cast => cast.ability.guid === ACTIONS.UPHEAVAL.id).length
+			const numOnslaughts = ir.casts.filter(cast => cast.ability.guid === ACTIONS.ONSLAUGHT.id).length
 
 			return {
 				key: ir.start,
 				title: {
 					content: <Fragment>
 						{this.parser.formatTimestamp(ir.start)}
-						&nbsp;-&nbsp;{numGcds} GCDs
+						&nbsp;-&nbsp;
```
Comment (MikeMatrix) [inline src/parser/jobs/war/InnerRelease.js]: Delet `&nbsp;` :shipit:

File: src/parser/jobs/war/InnerRelease.js [hunk 3]
```diff
@@ -100,19 +105,34 @@ export default class InnerRelease extends Module {
 			}))
 		}
 
-		if (this._missedGcds) {
+		if (this._missedGcds && !this._isRushing) {
```
Comment (ackwell) [inline src/parser/jobs/war/InnerRelease.js]: This is only checking the rush right during the `complete` event, and  skipping the entire suggestion if it is.

I'd probably move the rush check to the IR calculations itself, and skip counting missed GCDs if the rush is active.

Jokes just got up to that bit, you're already skipping the count if rushing. Can probably just remove this bit of the conditional then.

**Pull request merged**